go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/filecoin-project/specs-actors
 go 1.13
 
 require (
-	github.com/filecoin-project/go-address v0.0.0-20200107215422-da8eea2842b5
+	github.com/filecoin-project/go-address v0.0.2-0.20200218010043-eb9bb40ed5be
+	github.com/filecoin-project/go-amt-ipld v1.0.0
```

Comment:
This is probably an error, please revert.

Reply:
Yep, the wrong amt package snuck in through goimports trying to be smart.